sdk/src/main/java/com/silanis/esl/sdk/builder/PackageBuilder.java

```diff
@@ -26,6 +26,7 @@
     private SenderInfo senderInfo = null;
     private DocumentPackageAttributes attributes;
     private Visibility visibility;
+    private String timezoneId;
```

Comment:
I can't understand why a package should have a timezone. Is not the time zone necessary for the user? Please link the Acceptance Criteria of the relevant ticket.

Reply:
This is one of the major ideas behind the solution we discussed and agreed upon a few months back when we prepared the overall design for timezone support implementation. There was even a whole user story to implement this, please refer to https://jira.vasco.com/browse/PB-12568

Currently the API allows for retrieving/setting the timezone on a package, and that's why we're adding it to the SDK as well.